### ISSUE TITLE
Add .egg archive uncompress Plz.

### ISSUE THREAD

**Issue by redrug** [Bioruebe/UniExtract2#164]
Could you adding the .egg archive uncomress in the new version?
https://www.altools.co.kr/download/alzip.aspx
On the bottom of web page is the "UnEGG" tools,Thanks.

**Comment by vatterspun** [comment]
More background: https://en.wikipedia.org/wiki/EGG_(file_format)

In my research, I don't get the sense that it's a widely used compression format (outside of Korea) nor that it's better or worse than existing tools.  Still, assuming the decoder is free and small, I guess why not?

In the meantime, users coming across this thread looking for an EGG decompression program should probably look to BandiZip.

**Comment by redrug** [comment]
> More background: https://en.wikipedia.org/wiki/EGG_(file_format)
> 
> In my research, I don't get the sense that it's a widely used compression format (outside of Korea) nor that it's better or worse than existing tools. Still, assuming the decoder is free and small, I guess why not?
> 
> In the meantime, users coming across this thread looking for an EGG decompression program should probably look to BandiZip.

Yes...EGG file format was used by ALZIP (the Korea company) only.Maybe UE2 won't add the EGG support if  no more request,I wish UE2 can provide a interface to add plugins by user himself in the future release.Some times add a manual extraction mode will be more efficiency compare to auto scan mode when user know the target format and special program to extract it. @Bioruebe

**Comment by zufuliu** [comment]
Beware that .egg file extension is also used by Python setuptools, which is just a plain zip file.

https://setuptools.readthedocs.io/en/latest/formats.html#eggs-and-their-formats

**Comment by mrsshr** [comment]
@zufuliu 
korean egg file format is not python's egg file format.
this is a proprietary file compression file format of a korean software company.

[Bandizip](https://en.bandisoft.com/bandizip/) can handle this file format.
They also provide a command line version.
